lib/chef/file_access_control/windows.rb

```diff
@@ -128,7 +128,7 @@ def securable_object
       end
 
       def should_update_dacl?
-        return true unless ::File.exists?(file)
+        return true unless ::File.exists?(file) || ::File.symlink?(file)
```

Comment:
It *does* now, and we still want to be able to update the dacl of a symlink that goes nowhere (because it could have a file dropped in later).